Resources/ServerInfo/Funkystation/Guidebook/Command/CommandTrainingManual.xml

```diff
@@ -73,27 +74,36 @@
 
   - [bold]Revenants[/bold]
 
   Some former crew mates may find it hard to move on after an untimely death caused by their own negligence.
-  Nanotrasen recommends enlisting the Chaplain as an exorcist, without letting the living crew know about any prior deaths on the station.
+  Nanotrasen recommends enlisting the Chaplain as an exorcist, and follow his guidance as the Chaplains abilities and faith are one of the only solutions against this thaumaturgical threat.
 
   - [bold]Heretics[/bold]
 
-  Some individual crewmates may find ways to use more magical powers. These powers are heretical and not sanctioned by Nanotrasen. You should try to neutralize their powers while still keeping their work contract intact.
-  The Head of Security should consider only divulging information regarding the specific case, such as heretics being able to teleport with a magical focus.
+  Some individual crewmates may find ways to use magical powers. These powers are heretical and not sanctioned by Nanotrasen. You should try to neutralize the user while keeping station damage to a minimum.
+  The Head of Security should consider only divulging information regarding the specific case, such as heretics being able to teleport with a magical focus. 
+  In the event a Heretic goes on a full offensive, it is recommended to fully arm security, warn the crew and keep imporant members such as command far away.
 
   - [bold]Cults[/bold]
 
   In contrast, cults are highly dangerous, organized groups of magical power.
   The Head of Security may consider telling all the security force and the chaplain about a cult presence.
-  The chaplain, in particular, should be protected and made aware of any events.
+  The chaplain, in particular, should be protected and made aware of any events as their holy water and bible can destroy rifts.
 
   - [bold]Zombies[/bold]
 
   In the event of a zombie infection, the Chief Medical Officer and their associated medical team should be made the highest priority on the station.
   Due to the nature of zombie infections, there is no need to withhold any information about them. The station alert level should be set to Violet and go into an emergency state.
+
+  - [bold]Bingles[/bold]
+
+  Bingles are a horrible little species of two legged blue blob freaks that run around and steal anything that isn't anchored down (And stuff that is!) to throw into their disgusting vile pit that grows with more stuff trown in.
+  The only refuge from bingles is immediate extermination with extreme force. If bingles are allowed to grow to a size that can accomodate crew into the bingle hole, it is a disasterous threat.
+  Bingles should be unveiled immediately as their bite carries a sedative sting that can easily knock a crew member down and feed them into the vomit-inducing pit.
+  Stay as far as possible from Bingles.
```

Comment:
```suggestion
Bingles are a hostile, bipedal blue organism known for the unauthorized removal of station assets. Stolen materials are deposited into a central biological containment pit, which increases in size and capacity as additional items are introduced.

If left unattended, Bingles present a significant operational hazard. Continued growth of the containment pit may result in widespread station damage and the potential abduction of personnel into the pit itself.

Immediate identification and neutralization are strongly advised to prevent escalation, loss of station assets, and unnecessary personnel risk. Upon detection, avoid all unnecessary proximity and initiate extermination protocols using maximum authorized force.
```

This sounds more corporate.